handbook/spikes.md

```diff
@@ -70,12 +70,12 @@ First of all, it's completely normal to find public speaking and giving presenta
 
 ## :crown: Research presentation Hall of Fame
 
-### [FAC16: Asynchronous JS](https://github.com/FAC-Sixteen/research/blob/master/week2/Asynchronous%20JS.md)
+### [FAC19: Debugging network requests](https://github.com/fac19/research/blob/master/week2/Debugging-network-requests.md)
```

Comment:
I feel like there's a conflict of interest here 😆

Reply:
I caught myself out - I wanted a more recent example than FAC16 but couldn't find a better one this early on, so updated ours to include audience participation. Bit of a workaround but fine.

Reply:
No it's good, was just joking 🙃